go/enclave/rollupchain/rollup_chain.go

```diff
@@ -444,12 +440,20 @@ func (rc *RollupChain) SubmitBlock(block types.Block, isLatest bool) common.Bloc
 	}
 
 	common.TraceWithID(rc.nodeID, "Update state: b_%d", common.ShortHash(block.Hash()))
-	blockState, _, newLogs := rc.updateState(&block)
+	blockState := rc.updateState(&block)
 	if blockState == nil {
 		return rc.noBlockStateBlockSubmissionResponse(&block)
 	}
 
-	encryptedLogs, err := rc.subscriptionManager.GetSubscribedLogsEncrypted(newLogs, blockState.HeadRollup)
+	logs := []*types.Log{}
```

Comment:
Just a style thing but because Gophers seem to hate `else` blocks, I'd probably have gone:
```
	logs, found := rc.storage.FetchLogs(block.Hash())
	if !found {
		log.Error("Could not retrieve logs for stored block state. Returning no logs")
		logs = []*types.Log{}
	}
```